spinalcordtoolbox/scripts/sct_dmri_display_bvecs.py

```diff
@@ -140,4 +221,3 @@ def main(argv=None):
 if __name__ == "__main__":
     init_sct()
     main(sys.argv[1:])
-
```

Comment:
Would you like to try writing a test for this script? There's an existing test in `unit_testing/cli/test_cli_dmri_display_bvecs.py` that you can base the new test off of.

Also, if you want to try something new while writing the test, I've just discovered that `matplotlib` provides the [`@image_comparison` and `@check_figures_equal`](https://matplotlib.org/stable/devel/testing.html#writing-an-image-comparison-test) decorators. I'm really curious to try this out, so I'd be happy to try writing a test for this too. Either way!

Reply:
To be honest, I have only basic experience with writing tests. But definitely I would like to learn it more deeply! So, we can collaborate on that.

Reply:
> But definitely I would like to learn it more deeply! So, we can collaborate on that.

I'm glad to hear this! :tada: 

As a starting point, we have some internal documentation on test writing [here](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Testing). If anything is confusing about those pages, feel free to message on Slack! (I'm always looking to tweak the pages to make them easier to understand, so any questions are very helpful.)

Reply:
Actually, now that I think about it a little more... I take back my suggestion about `@image_comparison` and `@check_figures_equal`... I'm not sure if they're very useful in this instance.

This is a pretty self-contained script: No other SCT code depends on it, and it doesn't really depend on any SCT code. So, the only time the image contents are likely to change is if we're making an intentional change... in which case, testing against the image contents isn't really useful.

I think it might be good enough to just check for the existence of `bvecs.png` (like what is done in the existing test).

Reply:
@valosekj I think the easiest way to add a test would be to just copy and paste the existing test, but change the name + the arguments to include bvals from the example dataset. 

In this case, we don't have to get too fancy, hehe. :sweat_smile: